src/livery-interactive/LiveryInteractive.ts

```diff
@@ -0,0 +1,45 @@
+import { customElement, LitElement } from 'lit-element';
+
+import { LiveryBridge } from '../LiveryBridge';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'livery-interactive': LiveryInteractive;
+  }
+}
+
+@customElement('livery-interactive')
+export class LiveryInteractive extends LitElement {
+  bridge!: LiveryBridge;
+
+  /**
+   * Returns a promise with the SDK's current latency in seconds.
+   */
+  public getLatency(): Promise<number> {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Why is this necessary? As a rule, always specify why you're disabling lint rule as a comment, e.g: `// eslint-disable-next-line @typescript-eslint/no-unsafe-return -- because ..`